src/pages/Address/AddressComp.tsx

```diff
@@ -260,12 +278,38 @@ export const AddressTransactions = ({
 
   return (
     <>
-      <TitleCard title={`${i18n.t('transaction.transactions')} (${localeNumberString(total)})`} isSingle />
+      <TitleCard
+        title={`${i18n.t('transaction.transactions')} (${localeNumberString(total)})`}
+        className={styles.transactionTitleCard}
+        isSingle
+        rear={
+          <Radio.Group
+            className={styles.layoutButtons}
+            options={[
+              { label: i18n.t('transaction.professional'), value: 'professional' },
+              { label: i18n.t('transaction.lite'), value: 'lite' },
+            ]}
+            onChange={({ target: { value } }) => onChangeLayout(value)}
+            value={layout}
+            optionType="button"
+            buttonStyle="solid"
+          />
+        }
+      />
       <AddressTransactionsPanel>
+        {layout === 'lite' && !isMobile && (
```

Comment:
I think it would be easier to understand if the two layouts were separated, since they are essentially two different things. Forcing code reuse would reduce readability.

It could be written like this instead:
```tsx
{layout === 'lite' ? (
  <>
    {!isMobile && (
      <div className={styles.liteTransactionHeader}>
        <div>{i18n.t('transaction.transaction_hash')}</div>
        <div>{i18n.t('transaction.height')}</div>
        <div>{i18n.t('transaction.time')}</div>
        <div>{`${i18n.t('transaction.input')} & ${i18n.t('transaction.output')}`}</div>
        <div>{i18n.t('transaction.capacity_change')}</div>
      </div>
    )}
    {txList.map((transaction: State.Transaction) => {
      const { transactionHash } = transaction
      return <TransactionLiteItem address={addressHash} transaction={transaction} key={transactionHash} />
    })}
  </>
) : (
  txList.map((transaction: State.Transaction, index: number) => {
    const { transactionHash } = transaction
    return (
      <TransactionItem
        address={addressHash}
        transaction={transaction}
        key={transactionHash}
        circleCorner={{
          bottom: index === transactions.length - 1 && totalPages === 1,
        }}
      />
    )
  })
)}
```

This separates the two layouts to improve readability.